gdmt/.htaccess

```diff
@@ -0,0 +1,9 @@
+# This is the htaccess file for the w3id name for the Generic Dataset Metadata Template vocabulary
+
+RewriteEngine on
+
+# Rewrite for a term id
+RewriteRule (.*) https://bioportal.bioontology.org/ontologies/GDMT/?p=classes&conceptid=https://w3id.org/gdmt/$1
+
+# Rewrite for ontology id
+RewriteRule ^$ https://bioportal.bioontology.org/ontologies/GDMT/?p=classes
```

Comment:
Looks like the first rule is going to catch everything before this one is checked.